src/features/slackapi/slackapi-utils.ts

```diff
@@ -103,18 +104,29 @@ Have a great week!
    * Post a daily slack message to users
    *
    * @param dailyCombinedData list of combined timebank and slack user data
+   * @param timeRegistrations all time registrations after yesterday
    */
-  export const postDailyMessage = (dailyCombinedData: DailyCombinedData[]) => {
+  export const postDailyMessage = (dailyCombinedData: DailyCombinedData[], timeRegistrations: timeRegistrations[], dayOfWeek:number) => {
```

Comment:
Function should look something like this:
```
  export const postDailyMessage = (dailyCombinedData: DailyCombinedData[], timeRegistrations: timeRegistrations[], dayOfWeek:number) => {
    dailyCombinedData.forEach(user => {
      const { slackId, personId, expected } = user;

      const timeRegistration = timeRegistrations.find(timeRegistration => timeRegistration.person === personId);

      if (!timeRegistration) return;
      const onVacation = TimeUtilities.checkIfUserIsOnVacation(timeRegistration, expected);

      if (!onVacation && expected > 0) {
        try {
          client.chat.postMessage({
            channel: slackId,
            text: constructDailyMessage(user, dayOfWeek)
          });
        } catch (error) {
          console.error(`Error while posting slack messages to user ${user.name}`);
        }
      }
    });
  };
```

And the checkIfUserIsOnVacation something like this:
```
  /**
   * Checks if user is on vacation
   *
   * @param timeRegistration users time registration
   * @param expected expected time of user
   * @returns true if user is on vacation, otherwise false
   */
  export const checkIfUserIsOnVacation = (timeRegistration: timeRegistrations, expected: number): Boolean => {
    const { date, time_registered } = timeRegistration;
    const today = DateTime.now().toISODate();

    return date === today && time_registered === expected;
  };
```

Reply:
Done

Reply:
I tried this solution, but couldn't get it working so I used my solution